### DISCUSSION TITLE
Dextinsify.html saying "JavaScript blocked" when opened in chrome

### DISCUSSION THREAD

**Discussion by ronnae** [3kh0/ext-remover#920] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When I open the Dextensify html it says "Your are not running a Chromium-based browser" and my javascript is blocked, what can I do to fix it

**Comment by AshtonDavies** [comment]
Enable JavaScript.

**Reply by ronnae** [reply]
I can't that's the problem

**Reply by v4pl** [reply]
your school problely blocked it so just use skiovox

**Reply by AshtonDavies** [reply, marked as answer]
Maybe a faulty Dextensify page if you're not using one of the main addresses.
